### PR TITLE
Add ingress-nginx project for use of image promoter

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -523,8 +523,8 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
-    - jichenjc@cn.ibm.com
-    - sbueringer@gmail.com
+      - jichenjc@cn.ibm.com
```
Comment (dims) [inline groups/groups.yaml:526]: Can you please avoid these white space changes?

File: groups/groups.yaml [hunk 2]
```diff
@@ -1327,3 +1326,12 @@ groups:
...
       - jorge@heptio.com
+
+  - email-id: k8s-infra-staging-ingress-nginx@kubernetes.io
+    name: k8s-infra-staging-ingress-nginx
+    description: |-
+      ACL for pushing ingress-nginx artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - aledbf@gmail.com
```
Comment (dims) [inline groups/groups.yaml:1337]: the owners file has a couple of people more. can you please add them here?

**Comment by dims** [comment]
@aledbf a couple of nits, please take a look

File: groups/groups.yaml (hunk 2 above)
Comment (aledbf) [reply groups/groups.yaml:1337]: done

File: groups/groups.yaml (hunk 1 above)
Comment (aledbf) [reply groups/groups.yaml]: Sorry about. Done

**Comment by aledbf** [comment]
@dims please review again. 
Thanks

**Comment by spiffxp** [comment]
Currently blocked on https://github.com/kubernetes/k8s.io/issues/852

**Comment by aledbf** [comment]
@spiffxp thank you for the update

**Comment by spiffxp** [comment]
/approve
I'll leave this here any since it LGTM, I just checked and we're still blocked unfortunately

**Comment by spiffxp** [comment]
Updated groups and ran `./infra/gcp/ensure-staging-storage.sh ingress-nginx`

**Comment by aledbf** [comment]
@spiffxp thank you!

**Pull request merged**